frontend/apps/filemanager/filemanagershortcuts.lua

```diff
@@ -50,15 +45,25 @@ function FileManagerShortcuts:updateItemTable()
         })
     end
 
+    table.sort(item_table, function(l, r)
+        return l.text < r.text
+    end)
+    table.insert(item_table, 1, {
+        text = _("Add new folder shortcut"),
+        callback = function()
+            self:addNewFolder()
+        end,
+    })
+
     -- try to stay on current page
-    local select_number = nil
+    local select_number
 
     if self.fm_bookmark.page and self.fm_bookmark.perpage then
         select_number = (self.fm_bookmark.page - 1) * self.fm_bookmark.perpage + 1
     end
 
     self.fm_bookmark:switchItemTable(nil,
-                                     item_table, select_number)
+        item_table, select_number)
```

Comment:
Only ended up in my diff because it was with the version with the second item_table.
```suggestion
                                     item_table, select_number)
```